pdsfile/pdsfile.py

```
@@ -4917,14 +4900,14 @@ def dirpath_and_prefix_for_checksum(self):
         """
 
         if self.archives_:
-            dirpath = ''.join([self.root_, self.archives_, self.voltype_,
+            dirpath = ''.join([self.root_, self.archives_, self.bundletype_,
```

Ruff (0.13.3)

Consider `f'{self.root_}{self.bundletype_}{self.bundleset_}'` instead of string join

Replace with `f'{self.root_}{self.bundletype_}{self.bundleset_}'`

(FLY002)